app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user, @comment.post)
+    else
+      render :create
+    end
+  end
+end
```

Comment:
- [ ] As I mentioned above Your project run well and but I am not able to create a user from the rails console and I can't see the forms created which is required for this project :cry: Please fix your database and migration and also work on the different method that makes you app work as expected and make sure to populate your database in order to see the form created and your project fit the given [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES). Also make sure that we are able to comment a post and also like it :+1: